src/chapters/booleans-and-conditionals/exercises.rst

```diff
@@ -72,7 +71,7 @@ Part B: Logical Operators
 -------------------------
 
 Use the logical ``and``, ``or``, and ``not`` operators in the following
-exercises. Once again, if you are not assigned to a repl.it classroom or Trinket course, 
+exercises. Once again, if you are not assigned Trinket course, 
```

Comment:
There isn't a problem with this line, but I think you missed a repl.it classroom reference back under Part A.

Reply:
I did and fixed